server/routes/commentsRoutes.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const validateToken = require("../middleware/validateTokenHandler");
+
+const router = express.Router();
+
+const {
+  addComment,
+  deleteComment,
+  getAllCommentsForBlog,
+} = require("../controllers/commentsController");
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider updating require style imports</b></div>
<div id="fix">

Consider updating the `require()` style imports to ES6 `import` statements for better TypeScript compatibility and modern JavaScript practices. Similar issues found at lines 1, 2, and 10.

For example:
- `const express = require("express")` → `import express from "express"`
- `const { addComment } = require(...)` → `import { addComment } from "..."`
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<blockquote>Check the AI-generated fix before applying</blockquote>
<div id="code">


```suggestion
import express from "express";
import validateToken from "../middleware/validateTokenHandler";
import {
  addComment,
  deleteComment,
  getAllCommentsForBlog,
} from "../controllers/commentsController";

const router = express.Router();
```

</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Divyanshu9822/blog-mern-app/pull/22#issuecomment-2544587479>#245782</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged